src/App/ProjectDetail/ProjectDetailController.java

```diff
@@ -184,6 +184,18 @@ private void onreleaseID(KeyEvent keyEvent) {
     private void showDays(ActionEvent event) throws IOException {
         getEsti_time();
```

Comment:
```getEsti_time()``` is used to display number of days between Start Date and End Date in  ```JFXTextField esti_time```
We should call this method only if the end date is valid i.e. equal or greater than start date.